src/ffi.rs

```diff
@@ -126,11 +248,73 @@ pub unsafe extern "C" fn router_remove_matcher(
     router.remove_matcher(priority, uuid)
 }
 
+/// Execute the router with the context.
+///
+/// # Arguments
+///
+/// - `router`: a pointer to the [`Router`] object returned by [`router_new`].
+/// - `context`: a pointer to the [`Context`] object.
+///
+/// # Returns
+///
+/// Returns `true` if the router was executed successfully,
+/// `false` for no match found.
```

Comment:
I think the "execution" is always successful, otherwise we panic. I think the boolean returning from this interface is something like an execution result.

Reply:
Fixed